controllers/pvc-mutator/sc-label/sync.go

```diff
@@ -0,0 +1,70 @@
+package sclabel
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/storageos/api-manager/internal/pkg/provisioner"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// LabelSetter is responsible for setting labels
+// based on given StorageClass's parameters.
+type LabelSetter struct {
+	client.Client
+	log logr.Logger
+}
+
+// NewLabelSetter returns a new PVC label sync mutating admission
+// controller.
+func NewLabelSetter(k8s client.Client) *LabelSetter {
+	return &LabelSetter{
+		Client: k8s,
+		log:    ctrl.Log,
```

Comment:
I'd set the logger name here, rather than on on L36:

```
		log:    ctrl.Log.WithName("sc-param-sync"),
```
This format matches the other controllers, e.g. `node-label-sync`, `volume-label-sync`.